test/support/ws.ex

```diff
@@ -21,7 +22,7 @@ defmodule JellyfishWeb.WS do
 
   @impl true
   def handle_frame({:binary, msg}, state) do
-    send(state.caller, msg)
+    send(state.caller, ControlMessage.decode(msg))
```

Comment:
that's a nitpick but I would unpack outer ControlMessage here. In actual tests we will be able to do 

```elixir
    assert_receive  %PeerDisconnected{peer_id: ^peer_id, room_id: ^room_id}
```